kie-server-parent/kie-server-router/kie-server-router-proxy/src/main/java/org/kie/server/router/KieServerRouter.java

```diff
@@ -284,29 +290,28 @@ public void start(String host,
                                                                host,
                                                                sslContext);
         }
-
+        
+        if (!isHttpEnabled && !TLS_ENABLED) {
+            throw new IllegalStateException ("No port has been configured, review  "+ROUTER_PORT+ " and "+ROUTER_PORT_TLS+ " system properties");
```

Comment:
Nitpick:
The error is actually thrown in case the HTTP port is explicitly set to <=0 and system property `org.kie.server.router.tls.keystore` is not defined. The exception text could be misleading.
What about something like:
`HTTP listener was disabled (by setting HTTP port to 0 or lower ) and TLS wasn't configured, no listener is available to handle requests.`